flexmeasures/ui/views/logged_in_user.py

```diff
@@ -13,12 +13,12 @@ def logged_in_user_view():
     Basic information about the currently logged-in user.
     Plus basic actions (logout, reset pwd)
     """
-    num_assets_in_account = GenericAsset.query.filter(
-        GenericAsset.account_id == current_user.account_id
-    ).count()
+    number_of_assets_in_account = get_number_of_assets_in_account(
+        current_user.account_id
+    )
     return render_flexmeasures_template(
         "admin/logged_in_user.html",
         logged_in_user=current_user,
         roles=",".join([role.name for role in current_user.roles]),
-        num_assets=num_assets_in_account,
+        num_assets=number_of_assets_in_account,
```

Comment:
I think I would call the function right here, as we're not doing anything else with that number than passing it on. But that's a matter of opinion.